clustered/client/src/main/java/org/ehcache/clustered/client/internal/store/ClusteredStore.java

```diff
@@ -582,15 +651,19 @@ public void initStore(final Store<?, ?> resource) {
       clusteredStore.storeProxy.addInvalidationListener(new ServerStoreProxy.InvalidationListener() {
         @Override
         public void onInvalidateHash(long hash) {
+          Enum result = StoreOperationOutcomes.EvictionOutcome.SUCCESS;
+          clusteredStore.evictionObserver.begin();
```

Comment:
That feels _wrong_. There is technically no local eviction for the clustered tier. What happens with the invalidation valve will trigger eviction stats in the upper tier. So by accounting them as eviction in this tier as well we are double counting. I would remove the stats interactions here.